Sources/Basic/TerminalController.swift

```diff
@@ -79,7 +79,9 @@ public final class TerminalController {
             return nil
         }
 
-        width = TerminalController.terminalWidth() ?? 80 // Assume default if we are not able to determine.
+        // Assume default if we are not able to determine.
+        let terminalWidth = TerminalController.terminalWidth() ?? 80
+        width = terminalWidth == 0 ? 80 : terminalWidth
```

Comment:
We should try to avoid repeating the constant:

```swift
if let terminalWidth = TerminalController.terminalWidth(), terminalWidth > 0 {
    width = terminalWidth
} else {
    width = 80
}
```

Reply:
😅 Way better. 
Fixed